x-pack/test/reporting_api_integration/reporting_and_security/download_csv_dashboard.ts

```diff
@@ -400,62 +400,6 @@ export default function ({ getService }: FtrProviderContext) {
         };
         expect(body).to.eql(expectedBody);
       });
-
-      it(`Searches large amount of data, stops at Max Size Reached`, async () => {
```

Comment:
This test wasn't working right previously - since the `maxSizeReached` setting was removed from the test server, the search was no longer stopping where this test thought it was.